Assets/NpcConfig.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEngine;
+
+using static NpcSystem.NpcTypes;
+
+namespace NpcSystem
+{
+    sealed class NpcConfig : MonoBehaviour
+    {
+        [SerializeField , Tooltip("prefab for the weapon")] private GameObject WeaponPrefab;
+        [SerializeField, Tooltip("position for the weapon to spawn")] private Transform WeaponSpawnLocation;
+        [SerializeField , Tooltip("prefabs of the implemented animated characters")] private GameObject[] characters; 
+        [SerializeField , Tooltip("location for the animated character to spanw(NOT world position)")] private Transform charactersSpawn;
+        
+        //<summary>
+        //set everything before the character spawns
+        //</summary>
+        //<param name ="type"> will pass what kind of character will be spawned</param>
+        public void SetNpc(NpcType type)
+        {
+            Debug.LogWarning("skins not yet implemented");
```

Comment:
Deze warning kan beter in de `Start` of `Awake` gedaan worden met een check of de lijst leeg is of niet.